dev/com.ibm.ws.jaxrs.2.x.monitor/src/com/ibm/ws/jaxrs/monitor/REST_Stats.java

```diff
@@ -138,6 +180,31 @@ public void updateRT(long elapsed) {
         this.responseTime.addDataPoint(elapsed);
     }
 
+    
+    public void updateMinutePreviousMaximumDuration(long max) {
+        this.minutePreviousMaximumDuration.setCurrentValue(max);
+    }
+    
+    public void updateMinutePreviousMinimumDuration(long min) {
+        this.minutePreviousMinimumDuration.setCurrentValue(min);
+    }
+    
+    public void updateMinutePrevious(long min) {
```

Comment:
Use of `min` to represent both minimum and minute may be confusing to readers.

Reply:
Fixed  (as well as the `updateMinuteLatest` method)